### ISSUE TITLE
Compile Examples

### ISSUE THREAD

**Issue by dackdel** [wearenocomputer/ofxncKinect#2]
OSX and Linux user here. Working on windows for the first time and hence the question. How do I compile the examples? Ive got the rest right. A KinectV2, oF 0.10.1, VS 2017 + oF(all working great) and the KinectV2 SDK installed on a Win 10x64. Sorry for raising this as an issue, dont know how else to ask you this.

**Comment by wearenocomputer** [comment]
@dackdel you can not compile this on OSX or Linux. It is Windows only.

**Comment by dackdel** [comment]
I am on a windows 10 x 64. I am not on a OSX or Linux. I want to know how to compile it on windows. I am used to using a OSX/Linux and hence I have the question of how do I compile it using the Project Generator. How?

**Comment by wearenocomputer** [comment]
Make sure to use the *.props file
In Visual Studio use View > Other Windows > Property Manager
In the Property Manager right click on Debug or Release x64 and -> Add Existing Property Sheet an then choose the property sheet in the example

**Comment by dackdel** [comment]
I think in your example code you have named the library as ofxncKinect and not ofxNCKinect. I renamed the folder before adding it to addons folder. 
I still receive errors 19 of them when I try to compile. 
I will tell you the steps I followed:
0. Install Kinect 2 SDK
1. Install oF 0.10.1 with VS 2017 on a Windows 10 x64bit machine.
2. Download unzip and copy ofxNCKinect to addons folder
3. Copy "examples" folder from ofxNCKinect to apps>myApps folder. 
4. Run project generator on the examples folder and it gives me an error "missing ofxncKinect" so I rename the addon folder to ofxncKinect and the error goes away. Project Generator works and gives success output for the "examples" folder. 
5. Open the project in vs2017 and right click "Debug | x64 and Release | x64" and add existing property sheet called "Kinect64.props"
6. Build and it gives me 19 errors. 
![IMG_3142](https://user-images.githubusercontent.com/380951/54685429-ca8efc00-4b3c-11e9-9106-336fc5963823.jpg)

In line 19 of ofApp.h file in the "example" folder it is nCKinectCamera but everywhere else its ncKinectv2Core or ncKinect3dScene.
Also the #include "ncKinectCamera.h" is a small c.
